django_mysql/cache.py

```diff
@@ -224,15 +224,15 @@ def _base_set(self, mode, key, value, timeout=DEFAULT_TIMEOUT):
             if mode == 'set':
                 query = self._set_query
                 params = (key, value, value_type, exp)
-            elif mode == 'add':
+            else:  # mode = 'add'
```

Comment:
this one seems odd 🤔 wouldnt something more appropriate be

```
elif mode == 'add':
   # stuff
else:
  raise ValueError('invalid mode specified')
```

Reply:
Yeah you're right, even though it's internal only and will be a pain to get coverage on